packages/p2p/src/components/orders/order-table/order-table-sell-row.jsx

```diff
@@ -0,0 +1,51 @@
+import { Table }                  from 'deriv-components';
+import { localize }               from 'deriv-translations';
+import PropTypes                  from 'prop-types';
+import React                      from 'react';
+import { getFormattedDateString } from 'Utils/date-time';
+
+// TODO: [p2p-refactor-row-component] - Reuse row component that we have in other places
```

Comment:
The only change needed for the order buy and sell row to match is to pass a string with `Buy` or `Sell`? We can do that now?

Reply:
@oskar-binary This was more about us not having to create the `Row` component each time. But if you think it's a good idea to do that, we can keep it as is and I'll remove the TODO. What do you think?

Reply:
@oskar-binary BTW, as for these two components `BuyOrderRowComponent` and `SellOrderRowComponent`, the value placement for cells 3 and 4 change position too. But that's okay, I can merge them if you think it's better.

Reply:
After discussion, we will keep it as is and remove the TODO.